ros_buildfarm/templates/snippet/publisher_warnings.xml.em

```diff
@@ -34,7 +33,8 @@
       <isEnabledForFailure>false</isEnabledForFailure>
       <isAggregatingResults>false</isAggregatingResults>
       <isBlameDisabled>false</isBlameDisabled>
-      <isForensicsDisabled>false</isForensicsDisabled>
+      <skipPublishingChecks>false</skipPublishingChecks>
+      <publishAllIssues>true</publishAllIssues>
```

Comment:
`publishAllIssues` is an additional configuration for publishing checks for
all issues not just issues which are `new` as of the current build.
For now I've left this as true until we have more information about the
reliability of the reference build info with our current setup.